Makefile

```diff
@@ -1,9 +1,10 @@
 # Required executables
-ifeq (, $(shell which python3))
- $(error "No python3 on PATH.")
+ifeq (, $(shell which python))
+ $(error "No python on PATH.")
 endif
-ifeq (, $(shell which pipenv))
- $(error "No pipenv on PATH.")
+PIPENV_CMD := python -m pipenv
```

Comment:
We now assume that users have a proper `python` setup on path with `pipenv` installed. No more assumptions otherwise.